Description: Use canned version of Topcom example that sometimes runs
 out of memory.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1707
Last-Update: 2020-12-17

--- a/M2/Macaulay2/packages/Topcom.m2
+++ b/M2/Macaulay2/packages/Topcom.m2
@@ -350,11 +350,27 @@
     triangulation.  Notice that {\tt tri} is a triangulation of the 
     polytope which is the convex hull of the columns of $A$, which are 
     the only points allowed in the triangulation.
-  Example
-    A = transpose matrix {{0,3},{0,1},{-1,-1},{1,-1},{-4,-2},{4,-2}}
-    tri = {{0,1,2}, {1,3,5}, {2,3,4}, {0,1,5}, 
-        {0,2,4}, {3,4,5}, {1,2,3}}
-    isRegularTriangulation(A,tri)
+  CannedExample
+    i1 : A = transpose matrix {{0,3},{0,1},{-1,-1},{1,-1},{-4,-2},{4,-2}}
+
+    o1 = | 0 0 -1 1  -4 4  |
+         | 3 1 -1 -1 -2 -2 |
+
+                  2        6
+    o1 : Matrix ZZ  <--- ZZ
+
+    i2 : tri = {{0,1,2}, {1,3,5}, {2,3,4}, {0,1,5},
+             {0,2,4}, {3,4,5}, {1,2,3}}
+
+    o2 = {{0, 1, 2}, {1, 3, 5}, {2, 3, 4}, {0, 1, 5}, {0, 2, 4}, {3, 4, 5}, {1,
+         ------------------------------------------------------------------------
+         2, 3}}
+
+    o2 : List
+
+    i3 : isRegularTriangulation(A,tri)
+
+    o3 = false
   Text
     Setting debugLevel to either 1,2, or 5 will give more detail about
     what files are written to Topcom, and what the executable is.
